app/scripts/views/table.js

```diff
@@ -14,8 +14,8 @@ default class TableView extends Component {
     return (
       <Table responsive>
         <tbody>
-          {this.props.table.map(val => {
-            return <tr><td>{addr(val)}</td></tr>
+          {this.props.table.map((val, i) => {
+            return <tr key={i}><td key={i}>{addr(val)}</td></tr>
```

Comment:
After looking more in-depth into how react handles the :key:  property, I've determined that the `key` on the `td` element actually does nothing in this case, so I've removed it. If `td` was in another (nested) array, it would need a key, but it's not.

Regarding duplicate keys, it seems that keys only [need to be unique amongst it's siblings](https://facebook.github.io/react/docs/reconciliation.html#keys). I believe this is because react doesn't use the key we provide as-is, but rather uses it to generate a unique `data-reactid`. In a case where a parent and a child have the same key, they would still have different `reactid`'s. If we have a key value of `1`, a parent would have something like `.0.1:$1` and child would have `.0.1:$1.1`, so there wouldn't actually be any conflict.